docs/administration-guide/16-authenticating-with-saml.md

```diff
@@ -0,0 +1,34 @@
+## Authenticating with SAML
```

Comment:
we should also write up docs for JWT

Reply:
Yep, going to push JWT docs up in a minute.